dulwich/objects.py

```diff
@@ -1072,6 +1072,9 @@ def format_timezone(offset, unnecessary_negative_timezone=False):
             (sign, offset / 3600, (offset / 60) % 60)).encode('ascii')
 
 
+MAX_TIME = 4294967296  # 2**32
```

Comment:
Is this the same limit that C Git uses?

Reply:
I'm not certain of it.
Thus my question here https://github.com/jelmer/dulwich/pull/568#discussion_r148078638.